### DISCUSSION TITLE
Star Exchange

### DISCUSSION THREAD

**Discussion by ArielMAJ** [Schweinepriester/github-profile-achievements#659] [General]
Star my repo and leave your repo below for me to star it too.

https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by bkpecho** [comment]
Hello @ArielMAJ. I already starred ⭐ your repository. Feel free to star back mine :)

**My repository:** https://github.com/bkpecho/bkpecho

**Reply by ArielMAJ** [reply]
Tyvm! I starred you back!

**Reply by bkpecho** [reply]
Thank you!

**Reply by geamdev** [reply]
❤️

**Comment by peterbaccaro** [comment]


**Reply by ArielMAJ** [reply]
@peterbaccaro I starred you, please star my repo too: https://github.com/ArielMAJ/test.me

**Reply by dqanshi** [reply]
Hyy

**Comment by bensonarafat** [comment]
i have pls star mine https://github.com/codepymdev/grad

**Reply by ArielMAJ** [reply]
Already starred you @bensonarafat, waiting for you to star back!

**Reply by dqanshi** [reply]
Give to me vro

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰

**Comment by dqanshi** [comment]
Please follow my github account 🔥

**Reply by ArielMAJ** [reply]
@dqanshi follow me back too

**Comment by dqanshi** [comment]
Give me a star  github.com/dangerbots/dangercat

**Reply by ArielMAJ** [reply]
https://github.com/ArielMAJ/test.me

**Reply by ArielMAJ** [reply]
@dqanshi starred. Star me back too

**Comment by jaenfigueroa** [comment]
this is mine https://github.com/jaenfigueroa/JaenCraft, I return the number of stars is your repositories with more stars

**Reply by ArielMAJ** [reply]
@jaenfigueroa starred you. Star me back too

**Comment by YuZhangWang** [comment]
done
[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)

**Comment by Denisyasyuchenya** [comment]
done
https://github.com/Denisyasyuchenya/my-location-app

**Comment by Sriansh-raj** [comment]
> Star my repo and leave your repo below for me to star it too.
> 
> https://github.com/ArielMAJ/test.me Feel free to contribute as well, there are many somewhat "simple" issues.

Pls star mine also 😢 https://github.com/Sriansh-raj/GitHub-Achievements

i've starred yours

**Reply by ArielMAJ** [reply]
Done

**Comment by Rantoryu** [comment]
@ArielMAJ 
Done, please star my repo: https://github.com/Rantoryu/Personal_Website
I will star back anyone in the comment to this!

**Reply by ArielMAJ** [reply]
Done! Ty!
